### ISSUE TITLE
Strange window behavior with tile layouts

### ISSUE THREAD

**Issue by cagelight** [awesomeWM/awesome#1436]
```
awesome v4.0 (Harder, Better, Faster, Stronger)
 • Compiled against Lua 5.3.3 (running with Lua 5.3)
 • D-Bus support: ✔
 • execinfo support: ✔
 • RandR 1.5 support: ✔
 • LGI version: 0.9.1
```

Some applications (video players specifically) seem to not launch right with layouts that force a size.
gnome-mplayer starts transparent, and doesn't become visible until you mess with the window.
ffmplayer and vlc both start visible, but only showing a small video that doesn't take up the whole window.

There were no such issues with awesome 3.5.

**Comment by psychon** [comment]
This might be a duplicate of #1369. Can you check if this issue still occurs with latest git/master?
The fix should be this: https://github.com/awesomeWM/awesome/pull/1388/commits/3da4802adb8a79a604c5d75ae27b35c0cdad4174

**Comment by Elv13** [comment]
I can reproduce too (with VLC). It also happen with my own layout system, so the bug is deeper than that.

**Comment by cagelight** [comment]
I've just tried the latest git/master, and that does fix the issue. Now if only you guys could fix #1321 Awesome 4.0 would be perfect :stuck_out_tongue:

**Comment by Elv13** [comment]
> Now if only you guys could fix #1321 Awesome 4.0 would be perfect 😛

Ok, I will take a look later
